src/components/10-atoms/fieldset/story.js

```diff
@@ -16,24 +16,46 @@ storiesOf('Components|Atoms/Fieldset', module)
   })
   .add('Fieldset', () => {
     const error = text('error', '');
-    const horizontal = boolean('horizontal', false);
+    const options = {
+      None: null,
+      Stretch: 'stretch',
+      Horizontal: 'horizontal',
```

Comment:
I would name these "variables" without upper letter. So they are same to the value and a dev can see valid values immediately.

Reply:
This variables represent how the text is shown in the dropdown. So I wanted them to be uppercase. Bu ya it doesn't really matter. Will change them to lowercase, so that they look better in code.